diff --git a/node_modules/@expo/cli/build/src/start/server/metro/externals.js b/node_modules/@expo/cli/build/src/start/server/metro/externals.js
index 8341467..66dc827 100644
--- a/node_modules/@expo/cli/build/src/start/server/metro/externals.js
+++ b/node_modules/@expo/cli/build/src/start/server/metro/externals.js
@@ -19,7 +19,7 @@ function _interopRequireDefault(obj) {
 const NODE_STDLIB_MODULES = [
     "fs/promises",
     ...(_module.builtinModules || // @ts-expect-error
-    (process.binding ? Object.keys(process.binding("natives")) : []) || []).filter((x)=>!/^_|^(internal|v8|node-inspect)\/|\//.test(x) && ![
+    (process.binding ? Object.keys(process.binding("natives")) : []) || []).filter((x)=>!/^_|^(internal|v8|node-inspect)\/|\/|:/.test(x) && ![
             "sys"
         ].includes(x)
     ),
